general-information.md

```diff
@@ -168,11 +168,9 @@ Note that to handle out-of-order events, the validity of the prior-state shall n
 | `available`, `reserved`, `on_trip`, `non_operational`, `elsewhere` | `unknown`     | `missing`            | The vehicle is not at its last reported GPS location, or that location is wildly in error |
 | `available`, `reserved`, `on_trip`, `non_operational`, `elsewhere` | `unknown`     | `comms_lost`       | The vehicle is unable to transmit its GPS location or other status information |
 
-NOTES:
+The *State Machine Diagram* shows how the `vehicle_state` and `event_type` relate to each other and how vehicles can transition between states. See [Google Slides](https://docs.google.com/presentation/d/1Ar2-ju8YlddSsTATvQw4YjsSa5108XtidtnJNk-UAfA/edit) for the source file.
+![MDS State Machine Diagram](https://raw.githubusercontent.com/schnuerle/mobility-data-specification/ms-state-diagram/MDS-state-machine-diagram.svg)
```

Comment:
This URL points to your fork of the repo and the branch this PR is coming in on. I think a relative URL like `/MDS-state-machine-diagram.svg` should work?